### ISSUE TITLE
Example doesn't run

### ISSUE THREAD

**Issue by aquavitae** [ehazlett/interlock#118]
I cloned the repo, cd'd into the nginx example and run docker-compose up, but the interlock container hangs on `waiting for event stream to become ready`.  Changing the docker-compose command to `-D run` for debugging, I get this output:

```
$ docker-compose up     
Starting nginx_app_1
Recreating nginx_interlock_1
Recreating nginx_nginx_1
Attaching to nginx_app_1, nginx_interlock_1, nginx_nginx_1
app_1       | 2016/04/12 11:21:19 listening on :8080
interlock_1 | time="2016-04-12T11:21:19Z" level=info msg="interlock 1.1.0 (8a68c99)" 
interlock_1 | time="2016-04-12T11:21:19Z" level=debug msg="docker client: url=" 
interlock_1 | time="2016-04-12T11:21:19Z" level=debug msg="starting event handling" 
interlock_1 | time="2016-04-12T11:21:19Z" level=debug msg="event received: status=interlock-start id=0 type= action=" 
interlock_1 | time="2016-04-12T11:21:19Z" level=error msg="event stream fail; attempting to reconnect" 
interlock_1 | time="2016-04-12T11:21:19Z" level=info msg="waiting for event stream to become ready" 
interlock_1 | time="2016-04-12T11:21:19Z" level=debug msg="event stream not yet ready; retrying" 
interlock_1 | time="2016-04-12T11:21:20Z" level=debug msg="event stream not yet ready; retrying" 
interlock_1 | time="2016-04-12T11:21:21Z" level=debug msg="event stream not yet ready; retrying" 
interlock_1 | time="2016-04-12T11:21:22Z" level=debug msg="event stream not yet ready; retrying" 
interlock_1 | time="2016-04-12T11:21:23Z" level=debug msg="event stream not yet ready; retrying" 
interlock_1 | time="2016-04-12T11:21:24Z" level=debug msg="event stream not yet ready; retrying" 
interlock_1 | time="2016-04-12T11:21:25Z" level=debug msg="event stream not yet ready; retrying" 
interlock_1 | time="2016-04-12T11:21:26Z" level=debug msg="event stream not yet ready; retrying" 
interlock_1 | time="2016-04-12T11:21:27Z" level=debug msg="event stream not yet ready; retrying" 
interlock_1 | time="2016-04-12T11:21:28Z" level=debug msg="event stream not yet ready; retrying" 
interlock_1 | time="2016-04-12T11:21:29Z" level=debug msg="event stream not yet ready; retrying" 
interlock_1 | time="2016-04-12T11:21:30Z" level=debug msg="event stream not yet ready; retrying" 
interlock_1 | time="2016-04-12T11:21:31Z" level=debug msg="event stream not yet ready; retrying" 
interlock_1 | time="2016-04-12T11:21:32Z" level=debug msg="event stream not yet ready; retrying" 
^CGracefully stopping... (press Ctrl+C again to force)
Stopping nginx_nginx_1 ... done
Stopping nginx_interlock_1 ... done
Stopping nginx_app_1 ... done
```

Docker is definitely running and accessible:

```
$ docker info
Containers: 3
 Running: 0
 Paused: 0
 Stopped: 3
Images: 129
Server Version: 1.10.3
Storage Driver: devicemapper
 Pool Name: docker-8:1-3146722-pool
 Pool Blocksize: 65.54 kB
 Base Device Size: 107.4 GB
 Backing Filesystem: ext4
 Data file: /dev/loop0
 Metadata file: /dev/loop1
 Data Space Used: 10.7 GB
 Data Space Total: 107.4 GB
 Data Space Available: 36.63 GB
 Metadata Space Used: 19.67 MB
 Metadata Space Total: 2.147 GB
 Metadata Space Available: 2.128 GB
 Udev Sync Supported: true
 Deferred Removal Enabled: false
 Deferred Deletion Enabled: false
 Deferred Deleted Device Count: 0
 Data loop file: /var/lib/docker/devicemapper/devicemapper/data
 WARNING: Usage of loopback devices is strongly discouraged for production use. Either use `--storage-opt dm.thinpooldev` or use `--storage-opt dm.no_warn_on_loop_devices=true` to suppress this warning.
 Metadata loop file: /var/lib/docker/devicemapper/devicemapper/metadata
 Library Version: 1.02.121 (2016-03-26)
Execution Driver: native-0.2
Logging Driver: json-file
Plugins: 
 Volume: local
 Network: null host bridge
Kernel Version: 4.4.5-1-ARCH
Operating System: Arch Linux
OSType: linux
Architecture: x86_64
CPUs: 4
Total Memory: 7.708 GiB
Name: archlinux
ID: V2FE:ZF7C:5EYV:3O6L:77E3:MKEK:IOZI:BQ3Q:FNPE:4GYM:LUAP:D6RZ
```

**Comment by princemaple** [comment]
seeing the same with haproxy here

```
interlock_1 | time="2016-04-12T12:10:40Z" level=info msg="interlock 1.1.0 (8a68c99)"
interlock_1 | time="2016-04-12T12:10:40Z" level=debug msg="docker client: url="
interlock_1 | time="2016-04-12T12:10:40Z" level=debug msg="starting event handling"
interlock_1 | time="2016-04-12T12:10:40Z" level=debug msg="event received: status=interlock-start id=0 type= action="
interlock_1 | time="2016-04-12T12:10:40Z" level=error msg="event stream fail; attempting to reconnect"
interlock_1 | time="2016-04-12T12:10:40Z" level=info msg="waiting for event stream to become ready"
interlock_1 | time="2016-04-12T12:10:40Z" level=debug msg="event stream not yet ready; retrying"
interlock_1 | time="2016-04-12T12:10:41Z" level=debug msg="event stream not yet ready; retrying"
interlock_1 | time="2016-04-12T12:10:42Z" level=debug msg="event stream not yet ready; retrying"
interlock_1 | time="2016-04-12T12:10:43Z" level=debug msg="event stream not yet ready; retrying"
interlock_1 | time="2016-04-12T12:10:44Z" level=debug msg="event stream not yet ready; retrying"
interlock_1 | time="2016-04-12T12:10:45Z" level=debug msg="event stream not yet ready; retrying"
interlock_1 | time="2016-04-12T12:10:46Z" level=debug msg="event stream not yet ready; retrying"
interlock_1 | time="2016-04-12T12:10:47Z" level=debug msg="event stream not yet ready; retrying"
interlock_1 | time="2016-04-12T12:10:48Z" level=debug msg="event stream not yet ready; retrying"
interlock_1 | time="2016-04-12T12:10:49Z" level=debug msg="event stream not yet ready; retrying"
interlock_1 | time="2016-04-12T12:10:50Z" level=debug msg="event stream not yet ready; retrying"
```

**Comment by princemaple** [comment]
Oh cool, the haproxy config path is addressed in https://github.com/ehazlett/interlock/pull/113.

**Comment by aquavitae** [comment]
Just tried again on a brand new ubuntu 14.04 server - same issue.  I dont' see anything wrong with the config though.

**Comment by ehazlett** [comment]
Ok I'll take a look.

**Comment by ehazlett** [comment]
This is fixed in e55339e84.  Thanks for reporting!
